jsdojo/random.js

```diff
@@ -0,0 +1,11 @@
+(function () {
+    "use strict";
+    kintone.events.on('app.record.index.show', function(event) {
+        var button = document.createElement("button") // create the button element
+        button.innerHTML = 'Choose a random employee!' // adds HTML text to the button
+        button.onclick = function() { 
+            alert((_.sample(event.records)).Text.value);
```

Comment:
- Do you need to put `_.sample(event.records)` in `()`?
- I guess it would be an error if no records are showing in the list view.
`_.sample(event.records)` will return `undefined` when the number of records showing in the list view is 0.
Therefore, `_.sample(event.records)).Text.value` should be an error in that case.
Would you add the code for handling the case where there are no records showing in the list view?